manifests/agent/subscription.pp

```diff
@@ -0,0 +1,22 @@
+# @summary Add agent subscription
+#
+# @example
+#   sensu::agent::subscription { 'mysql': }
+#
+# @param subscription
+#   Name of the subscription to add to agent.yml, defaults to `$name`.
+# @param order
+#   Order of the datacat fragment
+#
+define sensu::agent::subscription (
+  String $subscription = $name,
+  String $order        = '50',
```

Comment:
Should this also be `String[1]` or perhaps `Integer`?

Reply:
I used `String[1]` because doesn't need to be Integer and all docs for datacat use a string and the default in datacat is the string `'50'`.